src/Workspaces/Remote/Razor/Microsoft.CodeAnalysis.Remote.Razor.ServiceHub.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.Remote.Razor.ServiceHub</RootNamespace>
-    <TargetFramework>net472</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
😕 It seems like this would result in a build failure, since `Microsoft.CodeAnalysis.Remote.ServiceHub` does not support netstandard2.0.

Reply:
Yup, you are right. Missed that one. Fixed.